src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -14,6 +14,74 @@
 
     public void moveRobot(Robot robot, int toX, int toY) {
         //your code
+        int startX = robot.getX();
+        int startY = robot.getY();
+        while (startX != toX && startY != toY) {
```

Comment:
try to simplify this solution
for example: we can turn left/right/up/down from any position if we need it. It is unnecessary to check in which position robot is now :)